db/src/main/resources/db/migration/V1_002__UC_datafeed_initial_format.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE dwp_uc_household (
+    id uuid not null primary key,
+    household_identifier varchar(50),
+    file_import_number integer,
+    award_date date,
+    last_assessment_period_start date,
+    last_assessment_period_end date,
+    pregnancy_flag varchar(1),
+    earnings_threshold_exceeded_flag varchar(1),
+    no_of_children_under_four integer,
```

Comment:
Do we also need number of children under 1, or is that just used/collected by us on the application?

Reply:
no_of_children_under_four is provided by the feed - that's the only reason it's there. We can use the dwp_uc_child table to count children.